test/unit/specs/components/DecisioningEngine/utils.spec.js

```diff
@@ -65,4 +66,12 @@ describe("DecisioningEngine:utils", () => {
       done();
     }, 20);
   });
+  it("should return the date of expiration", () => {
+    const retentionPeriod = 10;
+    const today = new Date();
+    const expectedDate = new Date(today);
+    expectedDate.setDate(expectedDate.getDate() - retentionPeriod);
```

Comment:
This test is potentially fragile if the second changes between this call and the call to getExpirationDate.